maven-java/kalix-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 version: "3"
 services:
   kalix-proxy:
-    image: gcr.io/kalix-public/kalix-proxy:1.0.34
+    image: gcr.io/kalix-public/kalix-proxy:1.1.0
     command: -Dconfig.resource=dev-mode.conf -Dkalix.proxy.eventing.support=google-pubsub-emulator
```

Comment:
@johanandren shouldn't we add a logback config for archetypes as well?

Reply:
Yes, I probably missed that, good catch, I'll PR 👍

Reply:
https://github.com/lightbend/kalix-jvm-sdk/pull/1464